jetty9-base/src/main/jetty-base/etc/gae-override-web.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<web-app 
+   xmlns="http://xmlns.jcp.org/xml/ns/javaee" 
+   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+   xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-app_3_1.xsd" 
+   version="3.1"> 
+
+
+<!-- 
+  This web.xml format file is an override file that is applied to the webapp AFTER
+  it has been configured by the default descriptor and the WEB-INF/web.xml descriptor.
+  It can be used to override init-params and/or add/change additional servlets, filters
+  and mappings.
+-->
+
+  <servlet>
+    <servlet-name>default</servlet-name>
+    <init-param>
```

Comment:
I think we need a comment here specifying what it's actually doing -- disallowing directory listing when there is no welcome file.